### DISCUSSION TITLE
Group 3 Project Feedback: Nudging Physical Activity with Wearables

### DISCUSSION THREAD

**Discussion by bbdaniels** [gui2de/ppol768-spring23#157] [Group Projects Feedback]
Group 3, this is a great topic lends itself very well to such a project. There are basically two issues I would like to engage here that need to be addressed in the next version. The first is the idea of market failure and research design; the second is data collection.

For the idea of "market failure", this is the area where the research idea needs the most work. As it is, your study compares people who get something for free (the gym membership) to people who don't. That is the "poor people are poor" problem -- the intervention is relaxing a resource constraint, but not positing a market failure. Why are people not going to the gym? Consider: a basic gym membership in many rural areas is like $10 a month, so you are effectively giving someone $25 over ten weeks. Is that $25 realistically the difference between someone going to the gym or not? If they want to go, would giving them $25 overcome the barriers that exist? Or are there more structural issues that _prevent people from making the choices they want_ -- that is really what I mean by market failure here.

At the same time, you probably want to reduce the scope -- 400 participants "from different regions" is very broad -- why not pick, say, one state, so you can focus on very comparable participants? You will also be able to collect data more easily, as well as probably focus your intervention in a more practical way based on structures of local economy. You can also do more functional interventions such as at town or county level -- therefore you may only need to enroll service providers, instead of enrolling all sorts of small firms who (probably) don't know who you are and may want nothing to do with you! Focus in on a setting where you can succeed and draw clear comparison groups of similar people -- don't over-prioritize "representativeness" at a scope in which the experiment won't work in the first place.

Finally, data. Lab data is super invasive and ethically challenging. How are you making sure everyone is going to the doctor every other week to get weighed, if they already are poor and don't have time to go to the gym? How are you monitoring gym visits, if not through (optional) surveys? How are surveys delivered? What if people who don't go to the gym also don't answer surveys? What will you do if your data indicates someone has serious undiagnosed health problems? While your data collection doesn't need to be like, completely feasible (all plans undergo changes as they meet reality), it should be at least plausible -- I worry that there are a few too many holes here to get good measurements.

In sum, I like the idea and background, and the actual intervention and measurement need to be made a bit more robust as you continue developing this idea. Great work and I can't wait to see where you take it!

**Comment by BeaLeydier** [comment]
Hi team

Great job! I really like the focus on employee healthcare and on rural America, which are both very specific and interesting topics to explore. Building off Ben's comments, I have a few questions to help you refine your project structure

1. What exactly *is* the intervention ? Your intervention section discusses more about how EWPs have been shown to have benefits, but what is your particular project proposing to do and study - do you want to study how we can increase uptake in EWPs when they exist (and if yes what intervention makes uptake more likely according to you)? do you want to study what happens when an EWP is offered to someone who didn't have one (and if yes why)? 

2. Could your study be an RCT? Once you have identified the specific aspect you want to study, is that something you could manage and deliver in a consistent manner to people who don't already have it? For example, if what you wanted to do is make EWPs cheaper - could you provide vouchers? If what you wanted to do is make EWPs more rewarding with something monetary to gain for participation - could you provide that monetary gain? If what you wanted to do is provide a consistent form of coaching in line with what has been shown to work - is that something you could provide (assuming you can hire and train coaches to deliver a specific program you design)? I think once you have designed more precisely what it is that you are trying to tackle to increase EWP participation and/or impact, you may find that it is an intervention you can deliver as an RCT and so just do that because it will give you a much stronger embedded design than observign people with and without that specific aspect of your intervention.

3. Are the subjects the companies or the employees? Who receives the interventions and whose behavior are you aiming to change and what outcomes are you measuring? 

4. On top of the questions Ben raised, specifically on data how are you going to collect lab data on your control group? Relatedly, is it possible that collecting/sharing/knowing data about your own health is something that is more likely to make you health conscious and healthy in and of itself, in which case how are you disentangling effects of your program and of the measurement itself?

Cheers!

**Comment by bbdaniels** [comment]
# Concept Note Review

Coming out of our discussions, this is a great improvement on the study design and I like it a lot. I think the measurement and estimation is very well built out and the heterogeneity approach is super interesting.

The biggest expansion you need to make is in Intervention and Sampling. If there are 64 campuses and you are recruiting 500 people, that is less than 10 per campus. If you have ONLY by-campus stratification, you have already basically run out of ability to stratify. (Also, where are all these people going for the site medical visits if there are just 10 participants at each site? You will need almost as many staff as participants...) Anyway -- I don't want you to worry enormously about this from a practicality perspective (ie, what happens if you have thousands of respondents who want what's basically a free $1000), but do take it seriously. What _are_ the "key demographic characteristics"? What counts as "prior wearable experience"? Is "90% of waking hours" meaning, they can have it off for 10% of time, or they have to have it on 90% of time? How can you tell "waking hours"?

For randomization, you have three basic stages, which need to be a bit more explicit. First, you have overall inclusion in the study. Let's just make it concrete. If you have, say, a thousand applicants at each campus, how many will you choose from each? Which ones? Exactly how many of each? Second, from there, how are you stratifying that into nudge/no-nudge? Is your 20% control really big enough (you will want to do power calculations depending on effect size, which I will introduce next week in class)? Third, if you want to use "Thompson Sampling ML" for targeting, I expect an explanation of what that means -- are you randomizing when people get nudges to some extent? Does this mean you are comparing within the nudge group on dimensions like frequency and alignment with existing habits?

Anyway -- this is really good and shows a super clear understanding of our discussion. If you expand out the sampling and randomization elements -- effectively, building out an exact idea of your regression along the lines of the models I am discussing in my lectures for the class -- you will have a very successful project pitch. Great work!
